neuraxle/base.py

```diff
@@ -801,7 +1233,9 @@ def __reversed__(self) -> 'BaseStep':
 
 
 class MetaStepMixin:
-    """A class to represent a meta step which is used to optimize another step."""
+    """
+    A class to represent a meta step which is used wrap a step, and add behavior to it.
```

Comment:
This class is so important. It would be good to have a big docstring with one or two examples.